packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -11204,28 +11194,30 @@ Generated by [AVA](https://ava.li).
       1. onPreBuild command from ./plugin                           ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-    Netlify configuration property "functions.node_bundler" value changed to 'zisi'.␊
-    Netlify configuration property "functions.external_node_modules" value changed to [ 'test' ].␊
-    Netlify configuration property "functions.ignored_node_modules" value changed to [ 'test' ].␊
-    Netlify configuration property "functions.included_files" value changed to [ 'test' ].␊
+    Netlify configuration property "functions.test.external_node_modules" value changed to [].␊
+    Netlify configuration property "functions.test.ignored_node_modules" value changed to [].␊
+    Netlify configuration property "functions.test.included_files" value changed to [].␊
     Netlify configuration property "functions.*.node_bundler" value changed to 'zisi'.␊
     Netlify configuration property "functions.*.external_node_modules" value changed to [].␊
     Netlify configuration property "functions.*.ignored_node_modules" value changed to [].␊
     Netlify configuration property "functions.*.included_files" value changed to [].␊
-    Netlify configuration property "functions.test.node_bundler" value changed to 'esbuild'.␊
```

Comment:
That specific log line disappeared since the value was already `esbuild`. This is good, i.e. non-mutations are not logged.